circle.yml

```diff
@@ -6,4 +6,9 @@ machine:
     OMNIAUTH_CLIENT_SECRET: 'omniauth_client_secret'
     OMNIAUTH_CLIENT_ID: 'omniauth_client_id'
     DEVISE_SECRET_KEY: 'devise_secret_key'
+    AWS_ACCESS_KEY_ID: 'aws_key'
```

Comment:
@Tuuleh We shouldn't need sensitive data in ENV for specs to run, but of course for CI to work we'll need certain keys set. 

I think I'll populate `test.rb` with the required ENV data for green specs. I'll then kill the above leaving you to manage the necessary variables on CircleCI for auto deployment.